code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -0,0 +1,246 @@
+			uniform = /obj/item/clothing/under/color/grey/glorf
+			shoes = /obj/item/clothing/shoes/sneakers/black
+			back = /obj/item/weapon/storage/backpack
+	..()
+
+/obj/effect/mob_spawn/human/hermit/Destroy()
+	new/obj/structure/fluff/empty_cryostasis_sleeper(get_turf(src))
+	..()
+
+//Broken rejuvenation pod: Spawns in animal hospitals in lavaland. Ghosts become disoriented interns and are advised to search for help.
+/obj/effect/mob_spawn/human/doctor/alive/lavaland
+	name = "broken rejuvenation pod"
+	desc = "A small sleeper typically used to instantly restore minor wounds. This one seems broken, and its occupant is comatose."
+	mob_name = "a translocated vet"
+	flavour_text = "<font size=3><b>W</b></font><b>hat...? Where are you? Where are the others? This is still the animal hospital - you should know, you've been an intern here for weeks - but \
+	everyone's gone. One of the cats scratched you just a few minutes ago. That's why you were in the pod - to heal the scratch. The scabs are still fresh; you see them right now. So where is \
+	everyone? Where did they go? What happened to the hospital? And is that <i>smoke</i> you smell? You need to find someone else. Maybe they can tell you what happened.</b>"
+
+//Prisoner containment sleeper: Spawns in crashed prison ships in lavaland. Ghosts become escaped prisoners and are advised to find a way out of the mess they've gotten themselves into.
+/obj/effect/mob_spawn/human/prisoner_transport
+	name = "prisoner containment sleeper"
+	desc = "A sleeper designed to put its occupant into a deep coma, unbreakable until the sleeper turns off. This one's glass is cracked and you can see a pale, sleeping face staring out."
+	mob_name = "an escaped prisoner"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper_s"
+	uniform = /obj/item/clothing/under/rank/prisoner
+	mask = /obj/item/clothing/mask/breath
+	shoes = /obj/item/clothing/shoes/sneakers/orange
+	pocket1 = /obj/item/weapon/tank/internals/emergency_oxygen
+	roundstart = FALSE
+	death = FALSE
+	flavour_text = "<font size=3><b>G</b></font><b>ood. It seems as though your ship crashed. You're a prisoner, sentenced to hard work in one of Nanotrasen's labor camps, but it seems as \
+	though fate has other plans for you. You remember that you were convicted of "
+
+/obj/effect/mob_spawn/human/prisoner_transport/New()
+	var/list/crimes = list("murder, but whatever they might say, you <i>know</i> that it was self-defense", \
+	"rape, but they're lying since you both agreed first", \
+	"larceny, but they looked loaded and you know that they could have just bought another one", \
+	"embezzlement, but there was plenty to go around and you know that it was barely a dent", \
+	"dereliction of duty, but you'd been having a rough day and you thought you might have caught Pierrot's throat", \
+	"kidnapping, but they were a Syndicate spy and you had conclusive evidence about it had they bothered to ask", \
```

Comment:
I feel like some of these are a bit specific and don't leave as much room for players to make things up